docs/examples/real_data.ipynb

```diff
@@ -1,6 +1,7 @@
 {
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    df_pequiv = pd.read_stata("C:/01-main/03-ac135963/Projekte/p2n/examples/data/pequiv.dta")
</pre><p> </p><p>Beispiele, die auf keinem anderen Computer funktionieren, sollten m.E. nicht in der Doku sein ^^</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mariuzka/pop2net/pull/88/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>